### ISSUE TITLE
stuck at Update Jessie/Kali Repo Public Key

### ISSUE THREAD

**Issue by Vector1199** [Screetsec/TheFatRat#734]
![Screenshot_2022-07-30_10_35_05](https://user-images.githubusercontent.com/30725702/181919256-870cf370-e730-4d2e-b836-8b5d47d3b8dc.png)

**Comment by Vector1199** [comment]
can anyone help me with this?

**Comment by Morsmalleo** [comment]
It's not waiting for you to type a response in is it???

Like `yes/y or no/n`??

**Comment by Vector1199** [comment]
no, I tried this with yes/no but it didn't work it just stuck  their

**Comment by Vector1199** [comment]
> It's not waiting for you to type a response in is it???
> 
> Like `yes/y or no/n`??

I installed fat rat in Garuda Linux which is based on Arch and configured the black arch repo on it fatrat is available on the black arch repo so no building issues dependency problems  everything working fine  just use Arch based Distros 😅

**Comment by peterpt** [comment]
That is related to slow kali repositories , leave it there running and it will work .
Fatrat before changing repositories it cleans the previous downloaded repositories data , then it installs what it needs and then when it changes again to user repository it will do an update like if it was its first time , kali repositories are extremely slow , it is not hanged in there , it is updating probably at 10k/s in the background .

**Comment by Vector1199** [comment]
> it is not hanged in there , it is updating probably at 10k/s in the background .

i actually tried on my kali vm and it took 3 hours
